PTMS/src/common/navigation/BottomNav.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import BottomNavigation from "@mui/material/BottomNavigation";
+import BottomNavigationAction from "@mui/material/BottomNavigationAction";
+import HomeIcon from "@mui/icons-material/Home";
+import NoteAddIcon from "@mui/icons-material/NoteAdd";
+import ManageAccountsIcon from "@mui/icons-material/ManageAccounts";
+import { Link } from "react-router-dom";
+import { useResponsive } from "../mediaQuery";
+
+export const BottomNav = () => {
```

Comment:
czemu to się nazywa `BottomNav`? w komponencie niżej wygląda jakby było użyte jako nawigacja w jakimś headerze z logo (nie odpalałem lokalnie apki, tak trochę zgaduję jak to wygląda :smile:)

Reply:
Trochę się zakręciłem z MUI i faktycznie nazwy nie końca trafione, dobra lekcja na przyszłość! 👌